src/audio/tone.c

```diff
@@ -530,7 +530,8 @@ static int tone_cmd_set_data(struct comp_dev *dev,
 	case SOF_CTRL_CMD_ENUM:
 		comp_info(dev, "tone_cmd_set_data(), SOF_CTRL_CMD_ENUM, cdata->index = %u",
 			  cdata->index);
-		compv = (struct sof_ipc_ctrl_value_comp *)cdata->data->data;
+		compv = (struct sof_ipc_ctrl_value_comp *)ASSUME_ALIGNED(cdata->data->data, 4);
```

Comment:
do we also need this here? https://github.com/thesofproject/sof/blob/main/src/audio/volume/volume.c#L673

Reply:
> do we also need this here? https://github.com/thesofproject/sof/blob/main/src/audio/volume/volume.c#L673

Yes, it's aligned and packed IPC data - I've updated other places too.

Reply:
Random thought, should we maybe make IPC_GET_CDATA so we don't have to update n files everytime we make a change like this going forward.